charts/refinery/templates/hpa.yaml

```diff
@@ -15,20 +15,20 @@ spec:
   maxReplicas: {{ .Values.autoscaling.maxReplicas }}
   metrics:
     {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    {{- if .Values.autoscaling.targetMemoryUtilizationPercentage }}
```

Comment:
Looks like the CPU if statement needs moved.

Reply:
Good spot, I've pushed a fix.